lxd/seccomp/cgo.go

```diff
@@ -8,4 +8,5 @@ package seccomp
 // #cgo CFLAGS: -Werror=implicit-function-declaration
 // #cgo CFLAGS: -Werror=return-type -Wendif-labels -Werror=overflow
 // #cgo CFLAGS: -Wnested-externs -fexceptions
+// #cgo pkg-config: libcap
```

Comment:
As these changes are in a separate seccomp package I think it would warrant these changes be in a separate commit explaining what is being changed and why.

Reply:
Have done!